src/main/java/unicash/ui/SummaryWindow.java

```diff
@@ -17,26 +23,32 @@
  */
 public class SummaryWindow extends UiPart<Stage> {
 
-    public static final String USER_GUIDE_URL = "https://ay2324s1-cs2103-t16-3.github.io/tp/";
-    public static final String SUMMARY_MESSAGE = "Here is your summary:";
+    public static final String SUMMARY_MESSAGE = "Expense Summary:";
+    public static final String SUMMARY_MESSAGE_WHEN_NO_TRANSACTIONS = "There are no transactions available";
 
     private static final Logger logger = LogsCenter.getLogger(SummaryWindow.class);
     private static final String FXML = "SummaryWindow.fxml";
+    private static final int MAX_DISPLAYED_CATEGORIES = 10;
+    private static final int MAX_DISPLAYED_YEAR_MONTHS = 12;
+    private static final YearMonth EARLIEST_YEAR_MONTH = YearMonth.now().minusMonths(MAX_DISPLAYED_YEAR_MONTHS);
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MMM");
```

Comment:
Could we maybe standardize this so it's `MMM-yyyy` since that's the general convention used for DateTime too